blazar/cmd/manager.py

```diff
@@ -32,13 +32,15 @@
 
 
 class ManagerServiceSingleton:
-    _instance = manager_service.ManagerService()
+    _instance = None
 
     def __new__(self, resource_type=None):
+        if not ManagerServiceSingleton._instance:
```

Comment:
What is the difference here? I can understand that deferring construction would lead to a lot of different behaviors.

Reply:
It worked fine in the API process previously, but the manager process creating the instance before the config was loaded, and so was only loading the single instance plugin.

Reply:
Aha! Makes sense.